api/query/cache/lru/lru.go

```diff
@@ -12,6 +12,9 @@ import (
 	"time"
 )
 
+// Verify LRU interface compliance in compile-time.
```

Comment:
This will happen automatically. When we address ireturn lint errors, it will make this clearer. You can get rid of this for now.